lib/tbot/botfs/fs_linux.go

```diff
@@ -434,15 +434,16 @@ func HasACLSupport() (bool, error) {
 //
 // We've encountered this being incorrect in environments where access to the
 // kernel is hampered e.g. seccomp/apparmor/container runtimes.
-func HasSecureWriteSupport() (bool, error) {
+func HasSecureWriteSupport() bool {
 	minKernel := semver.New(Openat2MinKernel)
 	version, err := utils.KernelVersion()
 	if err != nil {
-		return false, trace.Wrap(err)
+		log.WithError(err).Info("Failed to determine kernel version. It will be assumed secure write support is not available.")
```

Comment:
Do we only ever call this once? It's gonna be annoying if we end up accidentally calling this in a loop.

Reply:
Yup - this is only called on initialisation at the mo.